x-pack/plugins/ml/public/maps/layer_selector.tsx

```diff
@@ -36,7 +36,7 @@ export class LayerSelector extends Component<Props, State> {
     const typicalActual: MlAnomalyLayers = selectedOptions[0].value! as
       | 'typical'
       | 'actual'
-      | 'connected';
+      | 'typical to actual';
```

Comment:
Can we not cast this `as MlAnomalyLayers`?

Reply:
Updated to use an enum to be used everywhere 2321384960d2018fc3aa3e54ddf0d4e5c95ae80a